### PR TITLE
security.txt implementation

### PR DESCRIPTION
**Pull request by stiliyana-simeonova**
## What does this change?
The change supports the security.txt implementation as described [here](https://securitytxt.org/)
It aims to aid vulnerability reporting.
The change adds 2 files (similarly to humans.txt) that are served under 
/.well-known/security.txt
/.well-knonw/security.txt.asc

Note: The change is linked to PR in platform [here](https://github.com/guardian/platform/pull/1385)

## Does this change need to be reproduced in dotcom-rendering ?
Not sure

## Screenshots
NA

## What is the value of this and can you measure success?
Provides quick access to security point of contacts if vulnerability is discovered by third party

### Does this affect other platforms?
No

### Does this affect GLabs Paid Content Pages? Should it have support for Paid Content?
No

### Does this change break ad-free?
Cannot test but I would not expect this to break ad-free

### Does this change update the version of CAPI we're using?
No

### Accessibility test checklist
Not tested

### Tested
No option to test - would appreciate if someone could test it

### CODE REVIEW

**Review by shtukas — APPROVED** [review]

File: facia/conf/application.conf [hunk 1]
```diff
@@ -1,6 +1,8 @@
 
 # Caching for humans.txt
 "assets.cache./public/humans.txt"="public, max-age=900"
+"assets.cache./public/security.txt"="public, max-age=0"
+"assets.cache./public/security.txt.asc"="public, max-age=0"
```
Comment (katebee) [inline facia/conf/application.conf]: We could probably cache these for the same duration as `humans.txt`?

**Comment by katebee** [comment]
Minor (@shtukas):
I have a preference for giving the person who opens the PR control of merging it. This allows everyone to control when their changes 'go live' and also be involved in the deployment. 
Appreciate that this is not always practical and different approaches may be helpful or appropriate in some circumstances. 😸 

There is an excellent article on the benefits of DevOps here:
https://www.theguardian.com/info/developer-blog/2015/jan/05/delivering-continuous-delivery-continuously

**Comment by shtukas** [comment]
@katebee 
I agree, but I had already promised @stiliyana-simeonova that I would, otherwise I would not have. Thanks :)

File: facia/conf/application.conf (hunk 1 above)
Comment (stiliyana-simeonova) [reply facia/conf/application.conf]: I was thinking about it to be honest because chances that they will change frequently is very very slim

**Comment by stiliyana-simeonova** [comment]
Excited to see if this will work - still 404s

File: facia/conf/application.conf (hunk 1 above)
Comment (shtukas) [reply facia/conf/application.conf]: Let me make a PR for that then :)

Comment (shtukas) [reply facia/conf/application.conf]: https://github.com/guardian/frontend/pull/22366

**Pull request merged**